share/translations/keepassxc_en.ts

```diff
@@ -9649,15 +9689,15 @@ Example: JBSWY3DPEHPK3PXP</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>&lt;p&gt;If you own a &lt;a href=&quot;https://www.yubico.com/&quot;&gt;YubiKey&lt;/a&gt; or &lt;a href=&quot;https://onlykey.io&quot;&gt;OnlyKey&lt;/a&gt;, you can use it for additional security.&lt;/p&gt;&lt;p&gt;The key requires one of its slots to be programmed as &lt;a href=&quot;https://www.yubico.com/products/services-software/challenge-response/&quot;&gt;HMAC-SHA1 Challenge-Response&lt;/a&gt;.&lt;/p&gt;</source>
+        <source>Detecting hardware keys…</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Detecting hardware keys…</source>
+        <source>No hardware keys detected</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>No hardware keys detected</source>
+        <source>&lt;p&gt;If you own a &lt;a href=&quot;https://www.yubico.com/&quot;&gt;YubiKey&lt;/a&gt; or &lt;a href=&quot;https://onlykey.io&quot;&gt;OnlyKey&lt;/a&gt;, you can use it for additional security.&lt;/p&gt;&lt;p&gt;The key requires one of its slots to be programmed as &lt;a href=&quot;https://docs.yubico.com/yesdk/users-manual/application-otp/challenge-response.html&quot;&gt;HMAC-SHA1 Challenge-Response&lt;/a&gt;.&lt;/p&gt;</source>
```

Comment:
Why did you reorder this?
(Same for src/gui/entry/EditEntryWidgetSSHAgent.ui)

Reply:
I don't know. I just did `./release-tool i18n lupdate` because sth. yelled at me. I don't remember tbh. But I'm pretty sure this will be obsolete when rebasing. I made it a separate commit so I can remove it easily.

FTR I'm still willing to finish that PR. It's just that life happened since I startet it. ;-)